catalogue/webapp/components/WorkDetails/WorkDetails.tsx

```diff
@@ -645,18 +645,34 @@ const WorkDetails: FunctionComponent<Props> = ({
           />
         ))}
 
+        {/*
+         A genre may be simple or compound.
+
+         A simple genre contains just one concept in its concepts list,
+         whereas a compound genre may contain many.
+
+         In both cases, the first concept is the "important" one that 
+         should be used to link to a concepts page.
+
+         Compound genres behave more like contributors than subjects.
+         The additional information imparted by the subsequent concepts
+         are more relevant to Genre as it relates to the Work in question 
+         than the Genre as its own thing.
+         */}
         {work.genres.length > 0 && (
           <WorkDetailsTags
             title="Type/Technique"
-            tags={work.genres.map(g => {
+            tags={work.genres.map(genre => {
               return {
-                textParts: g.concepts.map(c => c.label),
-                linkAttributes: worksLink(
-                  {
-                    'genres.label': [g.label],
+                textParts: genre.concepts.map(c => c.label),
+                linkAttributes: {
```

Comment:
Should we add a condition here, like we do for subject & contributor links, to ensure there is an attached concept page?

And I _think_ it's good to keep the `source` (which is the "'work_details/genres'" bit) for analytics reasons but @taceybadgerbrook could confirm.

Reply:
I would go in the other direction, and remove that condition from subject/contributor – at this point, the API should always be returning an `id` on subjects, contributors, and genres.

(Separately on this bit, I think we want a ConceptLink component for constructing these links consistently, a la WorkLink and ItemLink, but that's not a blocker for this patch.)

Reply:
When I search and select a works page, in Kibana, I see source:works_page_result. This is as expected. When I click on a subject or contributor link that takes me to concepts, Kibana show source is empty. When I click on a link on the concepts page and go to a works page, Kibana shows source:works_page_result. 

I think this second source value is being inherited from the earlier works page view. It would be much better if source:concepts

Reply:
@taceybadgerbrook 

WRT: having no source on subject/contributor/genre links from Works, I think I can sort that out as part of this, IIUIC, it's just a few lines.

WRT: fixing works_page_result (is that actually works_search_result?) I can give it a different source for each section on the concept page (like links from a Work, with `work_details/contributors`, `work_details/genres`, `work_details/subjects` being possible sources for a link to a works search). Is that desirable, or just `concepts`

Reply:
Looking at that last part again, I. think this is outside the scope of this change, as it is a problem that already exists, and it looks like it needs fairly widespread modifications that aren't really associated with the genre thing.  I have raised a new issue for it.

https://github.com/wellcomecollection/wellcomecollection.org/issues/9682